pupil-spa/src/app/services/azure-queue/queue-storage-stub.ts

```diff
@@ -18,27 +17,16 @@ const queueMessageEncoder: IQueueMessageEncoder = {
 };
 
 export const queueStorageStub: IQueueStorage = {
-  createQueueServiceWithSas: () => queueService,
+  createQueueServiceWithSas: () => queueServiceStub,
```

Comment:
do you need this class?  would it not be cleaner to instantiate a mock of the interface inside the test?